### DISCUSSION TITLE
Update wiki to explain how clocks are actually displayed

### DISCUSSION THREAD

**Discussion by denishowe** [online-go/online-go.com#2847] [Q&A]
The wiki section https://github.com/online-go/online-go.com/wiki/Finding-&-Starting-a-Game#basic-time-controls-explained shows some "abbreviated display of time settings" like
**Fischer: 3m+ 20s up to 5m**
but my clock on my (correspondence) game is
**24h (3)**
and my opponent's is
**23h 42m (SD)**

Please could someone update the wiki to explain the clocks actually shown on the site?

**Comment by BHydden** [comment]
The game you are referring to is using byo-yomi not fischer time, maybe that's your confusion?

**Comment by denishowe** [comment]
Thanks for your reply but I wasn't looking for an explanation of those particular clock displays, which I just gave as examples.

Please can you update the wiki page I mentioned.  It is supposed to explain the abbreviations of the different clock types displayed on the site but doesn't match what is actually displayed.

Cheers

**Comment by BHydden** [comment]
That section explains what you see on the [play](https://online-go.com/play) page, not an active game. The wiki is accurate to how games are displayed there.
